installers/olm/Makefile

```diff
@@ -0,0 +1,133 @@
+	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.5.3)
+
+.PHONY: bundles
+bundles: ## Build OLM bundles
+bundles: check-version $(distros:%=bundles/%)
+
+# https://olm.operatorframework.io/docs/tasks/creating-operator-bundle/#validating-your-bundle
+# https://github.com/operator-framework/community-operators/blob/8a36a33/docs/packaging-required-criteria-ocp.md
+.PHONY: bundles/community
+bundles/community:
+	cd ../../config/manager/$(MODE)/ && $(KUSTOMIZE) edit set image psmdb-operator=$(IMAGE)
+	./generate.sh community
+
+	env operator-sdk bundle validate $@ --select-optional='suite=operatorframework'
+	env operator-sdk bundle validate $@ --select-optional='name=community' --optional-values='index-path=$@/Dockerfile'
+
+.PHONY: bundles/redhat
+bundles/redhat:
+	cd ../../config/manager/$(MODE)/ && $(KUSTOMIZE) edit set image psmdb-operator=$(IMAGE)
+	./generate.sh redhat
+	env operator-sdk bundle validate $@ --select-optional='suite=operatorframework'
+
+# The 'marketplace' configuration is currently identical to the 'redhat', so we just copy it here.
+.PHONY: bundles/marketplace
+bundles/marketplace:
+	cd ../../config/manager/$(MODE)/ && $(KUSTOMIZE) edit set image psmdb-operator=$(IMAGE)
+	./generate.sh marketplace
+	env operator-sdk bundle validate $@ --select-optional='suite=operatorframework'
+
+.PHONY: clean
+clean: clean-deprecated
+clean: ## Remove generated files and downloaded tools
+	rm -rf ./bundles ./projects ./tools
+
+.PHONY: clean-deprecated
+clean-deprecated:
+	rm -rf ./package
+
+.PHONY: help
+help: ALIGN=18
+help: ## Print this message
+	@awk -F ': ## ' -- "/^[^':]+: ## /"' { printf "'$$(tput bold)'%-$(ALIGN)s'$$(tput sgr0)' %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+.PHONY: install-olm
+install-olm: ## Install OLM in Kubernetes
+	env operator-sdk olm install
+
+.PHONY: tools
+tools: ## Download tools needed to build bundles
+
+tools: tools/$(SYSTEM)/jq
+tools/$(SYSTEM)/jq:
+	install -d '$(dir $@)'
+	curl -fSL -o '$@' "https://github.com/stedolan/jq/releases/download/jq-1.7.1/jq-$$(SYSTEM='$(SYSTEM)'; \
+		case "$$SYSTEM" in \
+			(linux-*) echo "$${SYSTEM/-amd/}";; (darwin-*) echo "$${SYSTEM/darwin-*/osx-amd64}";; (*) echo '$(SYSTEM)';; \
+		esac)"
+	chmod u+x '$@'
+
+tools: tools/$(SYSTEM)/kubectl
+tools/$(SYSTEM)/kubectl:
+	install -d '$(dir $@)'
+	curl -fSL -o '$@' 'https://dl.k8s.io/release/$(shell curl -Ls https://dl.k8s.io/release/stable-1.21.txt)/bin/$(OS_KERNEL)/$(OS_MACHINE)/kubectl'
+	chmod u+x '$@'
+
+# quay.io/operator-framework/operator-sdk
+tools: tools/$(SYSTEM)/operator-sdk
+tools/$(SYSTEM)/operator-sdk:
+	install -d '$(dir $@)'
+	curl -fSL -o '$@' 'https://github.com/operator-framework/operator-sdk/releases/download/v1.19.1/operator-sdk_$(OS_KERNEL)_$(OS_MACHINE)'
+	chmod u+x '$@'
+
+tools: tools/$(SYSTEM)/opm
+tools/$(SYSTEM)/opm:
+	install -d '$(dir $@)'
+	curl -fSL -o '$@' 'https://github.com/operator-framework/operator-registry/releases/download/v1.33.0/$(OS_KERNEL)-$(OS_MACHINE)-opm'
```

Comment:
can we update this? Also, shoud this be aligned with the version from operator-sdk?